.circleci/config.yml

```diff
@@ -145,13 +114,70 @@ workflows:
                 "no-warning-imports-guava",
                 "no-warning-imports-java-design-patterns",
                 "sonarqube",
+                "checkstyle-and-sevntu",
+                "spotbugs-and-pmd",
+                "site",
+                "release-dry-run",
+                "assembly-run-all-jar",
+                "no-error-test-sbe",
               ]
+              script-name: [ *validation_script ]
+              image-name: [ *custom_img ]
 
   idea:
     jobs:
       - run-inspections
 
   git-validation:
     jobs:
-      - no-merge-commits
-      - check-pull-number
+      - validate-with-script:
+          name: "git-no-merge-commits"
+          script-name: "./.ci/validation.sh"
+      - validate-with-script:
+          name: "git-check-pull-number"
+          script-name: "./.ci/validation.sh"
+
+  cli-validation:
+    jobs:
+      - validate-with-script:
+          name: "checkchmod"
+          script-name: "./.ci/checkchmod.sh"
+      - validate-with-script:
+          name: "releasenotes-gen"
+          script-name: "./.ci/releasenotes-gen.sh"
+
+  javac-validation:
+    jobs:
+      - validate-with-script:
+          name: "check-since-version"
+          script-name: "./.ci/validation.sh"
+      - validate-with-script:
+          name: "javac11"
+          script-name: "./.ci/validation.sh"
```

Comment:
Sorry this is over engineering.
We always tried to keep command to be copied and pasted in local terminal.
We need `./.ci/validation.sh javac11` to easily reference people what to run on line cal to reproduce on local. Keep in mind most frequent contributor is very very junior.

Btw, why we need javac11 is we already on javac11 ?

Reply:
> Btw, why we need javac11 is we already on javac11 ?

I remember some issue with certain input files for Java11 that we keep in non-compilable, each file should have reason in it

Reply:
> Sorry this is over engineering.

Would you prefer a `cmd` parameter, that has exact command to copy paste? I cannot name it `command` since this is reserved keyword.

Reply:
Yes, we need string value of whole command, to reuse it without brain.

Reply:
Done

Reply:
Also,

>Btw, why we need javac11 is we already on javac11 ?

```
➜  checkstyle git:(migrate-drone) ag -Q "Compilable by javac, but noncompilable" -A1
src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough2.java
10://Compilable by javac, but noncompilable by eclipse
11-package com.puppycrawl.tools.checkstyle.checks.coding.fallthrough;

src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/nofinalizer/InputNoFinalizerFallThrough.java
8://Compilable by javac, but noncompilable by eclipse
9-package com.puppycrawl.tools.checkstyle.checks.coding.nofinalizer;

src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough.java
10://Compilable by javac, but noncompilable by eclipse
11-package com.puppycrawl.tools.checkstyle.checks.coding.fallthrough;

src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/InputSemicolonBetweenImports.java
18: * Compilable by javac, but noncompilable by eclipse due to
19- * this <a href="https://bugs.eclipse.org/bugs/show_bug.cgi?id=425140">bug</a>

```

Reply:
>  I cannot name it command since this is reserved keyword.

Was able to work around this.